rest/taskana-rest-spring/src/test/java/pro/taskana/rest/TaskControllerIntTest.java

```diff
@@ -650,7 +720,7 @@ void testUpdateTaskOwnerOfReadyTaskSucceeds() {
         template.exchange(
             taskUrlString,
             HttpMethod.GET,
-            new HttpEntity<>(getHeadersForUser_1_2()),
+            new HttpEntity<>(restHelper.getHeadersUser_1_2()),
```

Comment:
method could be static and then imported statically ... same goes for the other getHeaderMethods. Maybe worth a small refactoring? :)